hello world
connected! true
testResourceConfiguration {
  "scheduled": true,
  "name": "./src/LoginPage/react/node.test.tsx",
  "ports": [
    3333
  ],
  "fs": "."
}
hello
hello
test failed AssertionError2: expected 'invalidEmail' to equal 'no_error'
    at file:///Users/adam/Code/kokomoBay/docs/node/src/LoginPage/react/node.test.mjs:86:14
    at Object.butThen (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:24:37)
    at Then.butThen (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:382:40)
    at Then.test (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:179:28)
    at Given.give (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:127:34)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Testeranto.givenThat.run (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:67:22)
    at async file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:240:14
    at async Object.receiveTestResourceConfig (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:265:29)
    at async NodeTesteranto.receiveTestResourceConfig (file:///Users/adam/Code/kokomoBay/docs/node/chunk-KHU6MVJS.mjs:497:47) {
  showDiff: true,
  actual: 'invalidEmail',
  expected: 'no_error',
  operator: 'strictEqual'
}
exiting gracefully with 1 failures.
